arm_j1_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: arm_joint1
    joint_speed: 2.0
    motor:
        id: 1
        init: 512
        min: 0
        max: 1023

arm_j2_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: arm_joint2
    joint_speed: 2.0
    motor:
        id: 2
        init: 512
        min: 0
        max: 1023
